src/scripts/user-widget/README.md

```diff
@@ -0,0 +1,38 @@
+# User widget
```

Comment:
The user widget is supposed to be pulled in as a git submodule, no?  This looks like a Git goof.